editoast/src/models/fixtures.rs

```diff
@@ -311,3 +311,28 @@ pub async fn create_work_schedules_fixture_set(
 
     (work_schedule_group, work_schedules)
 }
+
+pub struct MacroNodeFixtureSet {
+    pub project: Project,
+    pub study: Study,
+    pub scenario: Scenario,
+    pub timetable: Timetable,
+    pub infra: Infra,
+}
+pub async fn create_macro_node_fixtures_set(
```

Comment:
```suggestion
pub struct ScenarioFixtures {
    pub project: Project,
    pub study: Study,
    pub scenario: Scenario,
    pub timetable: Timetable,
    pub infra: Infra,
}

pub async fn scenario_fixtures(
```

nit: this is not actually related to nge nodes